backend/processing_image/VERSION

```diff
@@ -1 +1 @@
-3
+4
```

Comment:
do we still need to manually update this?

Reply:
Currently yeah